content/docs/operating/storage.md

```diff
@@ -253,7 +295,9 @@ that cannot be recovered is moved to a directory called `orphaned`
 you do not need it anymore.
 
 The crash recovery usually takes less than a minute. Should it take much
-longer, consult the log to find out what has gone wrong.
+longer, consult the log to find out what is going on. With increasing number of
+time series in the storage (archived or not), the re-indexing tends to dominate
+the recovery time and can take tens of minutes.
```

Comment:
Added it.

However, I believe this is just the normal madness with SSDs. LevelDB seems to be quite aggressive with small sync'd write if a lot of index entries are added. So after a while, the SSD locks up. I have provoked a giant crash recovery recently to test if it still works as expected. The re-indexing of 20M time series with a large SSD (i.e. probably with 16kiB page size) took 25m.